modules/quadkey/bigquery/sql/ST_GEOGPOINTFROMQUADINT.sql

```diff
@@ -0,0 +1,11 @@
+----------------------------
+-- Copyright (C) 2021 CARTO
+----------------------------
+
+CREATE OR REPLACE FUNCTION `@@BQ_PREFIX@@quadkey.ST_GEOGPOINTFROMQUADINT`(quadint INT64)
+RETURNS GEOGRAPHY AS (
+ST_GEOGPOINT(
+`@@BQ_PREFIX@@quadkey._TILE2LON`(`@@BQ_PREFIX@@quadkey.ZXY_FROMQUADINT`(quadint).x+0.5,`@@BQ_PREFIX@@quadkey.ZXY_FROMQUADINT`(quadint).z),
```

Comment:
Why is this 0.5 extent increase added? In any case I suggest to document it.

Reply:
You're right I add a comment.
The bbox coords is
```
[
(_TILE2LON(x),   _TILE2LAT(y),
(_TILE2LON(x),   _TILE2LAT(y+1),
(_TILE2LON(x+1), _TILE2LAT(y+1),
(_TILE2LON(x+1), _TILE2LAT(y)
]
```
To get the centroid i take
```
(_TILE2LON(x+0.5),_TILE2LAT(y+0.5))
```
I couldn't take the average of bbox coords because of the border effect at -180 deg +180 deg

Reply:
https://github.com/CartoDB/carto-spatial-extension/pull/130/commits/8bf4d559709ec771b4dbc9175faad1d36d9e3092

Reply:
Great!